brains/andalite.py

```diff
@@ -0,0 +1,107 @@
+import ast
+import asyncio
+import concurrent
+import json
+import pprint
+import queue
+import threading
+import websockets
+
+from sapiens import Sapiens
+
+class Andalite(Sapiens):
+    name = 'andalite'
+    lastSent = {}
+
+    def __init__(self, game, args=None):
+        Sapiens.__init__(self, game, args)
+        self.telepathy = Telepathy()
+
+    def Step(self):
+        self.sendMemory()
+        self.receiveBytes()
+        return Sapiens.Step(self)
+
+    def Event(self, evt):
```

Comment:
are Event, Path, Victory necessary here? (in fact, pretty sure Victory's gone from the base class)

Reply:
nope i'll clean 'em up